RealmSwift/Realm.swift

```diff
@@ -0,0 +1,562 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import Realm
+import Realm.Private
+
+/**
+A Realm instance (also referred to as "a realm") represents a Realm
+database.
+
+Realms can either be stored on disk (see `init(path:)`) or in
+memory (see `init(inMemoryIdentifier:)`).
+
+Realm instances are cached internally, and constructing equivalent Realm
+objects (with the same path or identifier) multiple times on a single thread
+within a single iteration of the run loop will normally return the same
+Realm object. If you specifically want to ensure a Realm object is
+destroyed (for example, if you wish to open a realm, check some property, and
+then possibly delete the realm file and re-open it), place the code which uses
+the realm within an `autoreleasepool {}` and ensure you have no other
+strong references to it.
```

Comment:
This paragraph isn't really true. They'll have the same backing `RLMRealm`, but you don't get the same `Realm` object every time.